salt/elife-xpub/init.sls

```diff
@@ -102,10 +102,8 @@ elife-xpub-database-available:
 
     cmd.run:
         - name: /usr/local/bin/wait-database.sh
-        - user: {{ pillar.elife.deploy_user.username }}
-        - cwd: /srv/elife-xpub
-        - user: {{ pillar.elife.deploy_user.username }}
         - cwd: /srv/elife-xpub
+        #- runas: {{ pillar.elife.deploy_user.username }}
```

Comment:
when uncommenting this the environment variables appear, at least with my `sudo` command, but then I start seeing this in the `postgres` container logs:
```
postgres_1     | Error: Database is uninitialized and superuser password is not specified.
postgres_1     |        You must specify POSTGRES_PASSWORD to a non-empty value for the
postgres_1     |        superuser. For example, "-e POSTGRES_PASSWORD=password" on "docker run".
postgres_1     | 
postgres_1     |        You may also use "POSTGRES_HOST_AUTH_METHOD=trust" to allow all
postgres_1     |        connections without a password. This is *not* recommended.
postgres_1     | 
postgres_1     |        See PostgreSQL documentation about "trust":
postgres_1     |        https://www.postgresql.org/docs/current/auth-trust.html
```

Reply:
https://github.com/docker-library/postgres/issues/681

Reply:
Vagrant testing seems to show that https://github.com/elifesciences/elife-xpub-deployment/pull/46 solves the problem even when running from a root shell.

Reply:
Vagrant testing seems to show that https://github.com/elifesciences/elife-xpub-deployment/pull/46 solves the problem even when running from a root shell.

Reply:
Vagrant testing seems to show that https://github.com/elifesciences/elife-xpub-deployment/pull/46 solves the problem even when running from a root shell.

Reply:
that PR has been merged now